src/components/Cards/Cards.tsx

```diff
@@ -1,53 +1,44 @@
 import { useState } from 'react';
-
 import { CardInterface } from '../../constants/settings';
 import Card from '../_commonComponents/Card/Card';
-import { getCards } from '../../constants/preparation';
 
-const cardList = getCards();
+interface CardsProps {
+	cardList: CardInterface[],
+	removedCards: Object,
```

Comment:
Avoid using `Object` type. It is too wide, apart from that this is a type of object constructor. Please use Card interface